sangbok/main.xml

```diff
@@ -4592,4 +4592,54 @@
 			hur innerligt jag älskar dig!
 		</p>
 	</song>
+	
+	<song name="I Californien"
+		  category="Esoterica">
+		<comment>
+			Vanligt förekommande på evenemang arrangerade av THS Centralt, till exempel Kårfullmäktiges och Fanborgens sittningar.
+		</comment>
```

Comment:
Comments are mainly intended for instructional purposes, not background info. In rare cases they may be placed after the song itself (so as not to detract from the important content) but in this case I think you can remove it completely.